examples/jsm/physics/RapierPhysics.js

```diff
@@ -31,14 +31,14 @@ function getShape( geometry ) {
 	} else if ( geometry.type === 'CylinderGeometry' ) {
 
 		const radius = parameters.radiusBottom !== undefined ? parameters.radiusBottom : 0.5;
-		const length = parameters.length !== undefined ? parameters.length : 0.5;
+		const length = parameters.height !== undefined ? parameters.height : 0.5;
```

Comment:
While testing I have notice the collisions for the cylinder meshes were incorrect. Turns out the wrong parameters were evaluated.